lib/cc/analyzer/issue_location_existence_validation.rb

```diff
@@ -0,0 +1,56 @@
+module CC
+  module Analyzer
+    class IssueLocationExistenceValidation < Validation
+      def valid?
+        if location["lines"]
+          valid_lines?
+        elsif location["positions"]
+          valid_position?(location["positions"]["begin"]) &&
```

Comment:
There's a `NoMethodError` risk if an engine does something really dumb like emit `positions: "foo"`. Maybe not worth addressing.